test/NuGet.Core.FuncTests/Dotnet.Integration.Test/PackCommandTests.cs

```diff
@@ -28,6 +28,54 @@ public PackCommandTests(MsbuildIntegrationTestFixture fixture)
             msbuildFixture = fixture;
         }
 
+        [PlatformFact(Platform.Windows)]
+        public void CreatePackageWithFiles()
```

Comment:
This change could be unit tested, and avoid slow integration tests.

Reply:
We had a conversation about why this bug even happened, and that exposed the fact that we didn't have a basic integration test that covered this fairly general case, and that this was a good opportunity to add an integration test instead. Interestingly, only the integration test included in this PR actually fails without the fix.

Reply:
I agree with Kat here. A unit test is not a replacement for the integration tests. 

I'd rather see us reevaluate existing integration tests that might be running slow, than caution against the speed of the integration tests.